alicloud/resource_alicloud_drds_instance.go

```diff
@@ -0,0 +1,174 @@
+package alicloud
+
+import (
+	"fmt"
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/drds"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"strconv"
+	"time"
+)
+
+func resourceAliCloudDRDSInstance() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAliCloudDRDSInstanceCreate,
+		Read:   resourceAliCloudDRDSInstanceRead,
+		Update: resourceAliCloudDRDSInstanceUpdate,
+		Delete: resourceAliCloudDRDSInstanceDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"description": &schema.Schema{
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validateStringLengthInRange(1, 129),
```

Comment:
I think description is optinal, although the api docs show it required. Please confirm it with drds developer.

Reply:
Precise intuition! Confirmed, it's optional. I also reported an issue to [drds](https://connect.aliyun.com/suggestion/40421?spm=5176.10697025.user_profile.4.15ac67f99HnrQN). Thanks.